src/plugins/uploader.js

```diff
@@ -0,0 +1,146 @@
+"use strict";
+
+const SocketIOFileUploadServer = require("socketio-file-upload/server");
+const Helper = require("../helper");
+const path = require("path");
+const fsextra = require("fs-extra");
+const fs = require("fs");
+const fileType = require("file-type");
+const readChunk = require("read-chunk");
+const crypto = require("crypto");
+const isUtf8 = require("is-utf8");
+const log = require("../log");
+
+const whitelist = [
+	"application/ogg",
+	"audio/midi",
+	"audio/mpeg",
+	"audio/ogg",
+	"audio/x-wav",
+	"image/bmp",
+	"image/gif",
+	"image/jpeg",
+	"image/png",
+	"image/webp",
+	"text/plain",
+	"video/mp4",
+	"video/ogg",
+	"video/webm",
+];
+
+class Uploader {
+	constructor(client, socket) {
+		const siofu = new SocketIOFileUploadServer();
```

Comment:
Can we rename this? It's pretty confusing.

Reply:
I disagree, it's pretty self-explanatory. It's what the SIOFU docs use, so I think thats a fine reason for keeping as-is. I don't care too much if you feel strongly.

Reply:
It's self explanatory after you go up and read the name and go "oh, I guess that's what it means" but it's really quick to forget what that means. But if you read it down you have 0 idea, no clue whatsoever what it is.

Reply:
My argument is I think anyone editing the file should have an idea of what it is, and this way we stay consistent with the docs

Reply:
I disagree.

I think that the "this should be slightly confusing so that you know what you are doing" is a bad trope that exists in software development that I think we should be getting rid of. It's an elitist view, imo, which is really not great for software development as a whole. We should be making things easier for people, not more difficult.

And there's no need to stay consistent with the docs, the docs have named something in a confusing way, let's get rid of that.

Reply:
Fixed